compiler/src/iree/compiler/Dialect/Util/IR/UtilBase.td

```diff
@@ -65,9 +65,7 @@ def Util_TiedOpStorageAttr :
   let constBuilderCall = "$_builder.getI64ArrayAttr($0)";
 }
 
-def Util_GlobalRefAttr : ConfinedAttr<FlatSymbolRefAttr, [
-  ReferToOp<"IREE::Util::GlobalOpInterface">,
-]>;
+defvar Util_GlobalRefAttr = FlatSymbolRefAttr;
```

Comment:
is ReferToOp going away? it seems nicer to have it declared here instead of needing all verifiers that may use this attribute to do it?

Reply:
Yes, as is it is unsafe and can lead to crashes. We need a new construct upstream to support this. I'm removing it to avoid issues until then.